src/Terminal/commands/rm.ts

```diff
@@ -57,6 +55,19 @@ export function rm(args: (string | number | boolean)[], server: BaseServer): voi
     for (const file of server.scripts.keys()) {
       if (file.startsWith(dir)) files.push(file);
     }
+    for (const file of server.textFiles.keys()) {
+      if (file.startsWith(dir)) files.push(file);
+    }
+    for (const file of server.messages) {
+      if (file.endsWith(".msg")) continue;
```

Comment:
This seems like the intention is to avoid deletion of messages, but in that case, why are you even iterating this array? Especially since `server.removeFiles` doesn't work on messages.

Reply:
The `server.messages` array contains the `.msg` files and the `.lit` files, so I iterate over the array and filter out all of the `.msg` files, since these can not be deleted.